helm/app-exporter/templates/configmap.yaml

```diff
@@ -16,7 +16,13 @@ data:
     service:
       collector:
         apps:
-          defaultTeam: "batman"
+          defaultTeam: "honeybadger"
```

Comment:
Just changing the default team. This is so all app metrics have a team even if the Chart.yaml annotation is missing.